lib/msf/ui/formatter/option_validate_error.rb

```diff
@@ -12,16 +12,32 @@ class OptionValidateError
         def self.print_error(mod, error)
           raise ArgumentError, "invalid error type #{error.class}, expected ::Msf::OptionValidateError" unless error.is_a?(::Msf::OptionValidateError)
 
-          if error.reasons.empty?
-            mod.print_error("#{error.class} The following options failed to validate: #{error.options.join(', ')}")
-          else
-            mod.print_error("#{error.class} The following options failed to validate:")
-            error.options.sort.each do |option_name|
-              reasons = error.reasons[option_name]
-              if reasons
-                mod.print_error("Invalid option #{option_name}: #{reasons.join(', ')}")
+          (0...error.options.length).each do |i|
+            # Assign module examples unless the value is not available within options as it wasn't created as an option object.
+            # example: value assigned directly to datastore without being created as an options object via `OptString.new` or similar.
+            # See spec/lib/msf/ui/console/command_dispatcher/exploit_spec.rb:279
+            option_examples = mod.options[error.options[i]].nil? ? [] : mod.options[error.options[i]].examples
+            option_value = mod.datastore[error.options[i]]
+
+            if error.reasons.empty?
+              if option_examples.empty? && (option_value.nil? || option_value.empty?)
```

Comment:
what data type is `option_value` here? if it's always a String you can do this
```suggestion
              if option_examples.empty? && option_value.blank?
```
if it's not always a string what other types can it be and do they all support `.empty?`

Reply:
Got that changed to `blank?`, thanks for calling that out 👍